### ISSUE TITLE
New Feature: Support using a TOML "dotfile" config file in the user's homedir

### ISSUE THREAD

**Issue by fadingNA** [peterdanwan/gimme_readme#33]
- Users want to be able to specify all of their options to your tool in a [TOML](https://toml.io/en/) formatted configuration file

Then, when the user runs your tool, it will search for a config file in the home dir and use those values. Or, if the user specifies values via args, those will override the default ones in the config.

**Comment by fadingNA** [comment]
https://github.com/peterdanwan/gimme_readme/pull/34

@peterdanwan

**Comment by peterdanwan** [comment]
Hello @fadingNA, just wanted to ask if you could do one more pull request with regards to fixing an import for the changes you made in PR #34:

https://github.com/peterdanwan/gimme_readme/blob/8875a333f5e9af2c5fe182dde422bc4bf1d059de/src/ai_config/geminiConfig.js#L4-L6

Specifically, line 5's import statement needs to have `.js` added to the end of the filename:

```js
import getTOMLFileValues from '../file_functions/getTOMLFileValues.js';
```

Otherwise, we get this error:
![error](https://github.com/user-attachments/assets/e37fa7ff-cb4c-4fa9-80b0-d6dbe20789a8)

**Comment by fadingNA** [comment]
@peterdanwan  fixed

**Comment by peterdanwan** [comment]
Closed by #35
